octobot_trading/enums.py

```diff
@@ -392,12 +416,12 @@ class ExchangeConstantsPositionColumns(enum.Enum):
     MARK_PRICE = "mark_price"
     LIQUIDATION_PRICE = "liquidation_price"
     BANKRUPTCY_PRICE = "bankruptcy_price"
-    UNREALIZED_PNL = "unrealised_pnl"   # unrealised_pnl on Bybit api and unrealized_pnl on exchange UI
-    REALISED_PNL = "realised_pnl"
+    UNREALIZED_PNL = "unrealized_pnl"  # unrealized_pnl on Bybit api and unrealized_pnl on exchange UI
+    REALIZED_PNL = "realized_pnl"
     CLOSING_FEE = "closing_fee"
-    QUANTITY = "quantity"
-    SIZE = "size"
-    NOTIONAL = "notional"
+    CONTRACT_SIZE = "contract_size"
+    POSITION_SIZE = "position_size"
+    POSITION_VALUE = "position_value"
```

Comment:
Why renaming here?

Reply:
I don't understand why we need to add a "position" prefix to something in a PositionEnum

Reply:
yeah you're right. I'll change the naming like that:
QUANTITY -> SINGLE_CONTRACT_VALUE
SIZE stays as it was
NOTIONAL -> VALUE